test/README.md

```diff
@@ -0,0 +1,48 @@
+# Torchaudio Test Suite
+
+## Structure of tests
+
+The followings are the categories of tests (and corresponding test modules) for `torchaudio`.
+
+### Purpose specific test suites
+
+#### Numerical compatibility agains existing software
+- [Librosa compatibility test](./test_librosa_compatibility.py)
+    Test suite for numerical compatibility against librosa.
+- [SoX compatibility test](./test_sox_compatibility.py)
+    Test suite for numerical compatibility against SoX.
+
+#### Result consistency with Pytorch framework.
+- [Torchscript consistency test](./test_torchscript_consistency.py)
+    Test suite to check 1. if an API is Torchscript-able, and 2. the results from Python and Torchscript match.
+- [Batch consistency test](./test_batch_consistency.py)
+    Test suite to check if functionals/Transforms handle single sample input and batch input and return the same result.
+
+### Module specific test suites
+
+The following test modules are defined for corresponding `torchaudio` module/functions.
+
+- [`torchaudio.datasets`](./test_datasets.py)
+- [`torchaudio.functional`](./test_functional.py)
+- [`torchaudio.transforms`](./test_transforms.py)
+- [`torchaudio.compliance.kaldi`](./test_compliance_kaldi.py)
+- [`torchaudio.kaldi_io`](./test_kaldi_io.py)
+- [`torchaudio.sox_effects`](test/test_sox_effects.py)
+- [`torchaudio.save`, `torchaudio.load`, `torchaudio.info`](test/test_io.py)
+
+### Others
```

Comment:
I would like to list all the test modules for the completeness, so that when there is a missing module from the list, we can say that it's our oversight.